arches_component_lab/src/arches_component_lab/widgets/ResourceInstanceMultiSelectWidget/components/ResourceInstanceMultiSelectWidgetEditor.vue

```diff
@@ -85,6 +89,10 @@ async function getOptions(page: number) {
     }
 }
 
+function refreshOptions() {
```

Comment:
I believe this is unnecessary? since in `onFiltered` the page number is a hardcoded `1`